src/main/java/dev/meyi/bn/modules/module/NotificationModule.java

```diff
@@ -26,17 +26,30 @@
 
 public class NotificationModule extends Module {
 
-  public static final ModuleName type = ModuleName.NOTIFICATION;
-  int longestXString = 0;
+  public transient static final ModuleName type = ModuleName.NOTIFICATION;
 
   public NotificationModule() {
     super();
   }
 
-  public NotificationModule(ModuleConfig module) {
-    super(module);
+  @Override
+  protected void draw(UMatrixStack matrices, float x, float y, float scale, boolean example) {
+    draw();
   }
 
+  @Override
+  protected float getWidth(float scale, boolean example) {
+    if(!"".equals(longestString)) {

```

Comment:
again

Reply:
It somehow manages to be null when loading but nothing that can't be solved